docs/02-app/01-building-your-application/02-data-fetching/03-server-actions-and-mutations.mdx

```diff
@@ -414,6 +414,8 @@ export function Signup() {
 
 > **Good to know:**
 >
+> - To display form errors in the user UI, you should return a **serializable object** instead of throwing an error (e.g. `throw new Error('...')`. Uncaught errors in Server Actions are considered to be an internal server error and the error message [will not be sent to the client](/docs/app/building-your-application/routing/error-handling#securing-sensitive-error-information) in production to avoid leaking sensitive information. See more information on [error handling](#error-handling) in Server Actions.
```

Comment:
```suggestion
> - To display form errors in the user UI, you should return a **serializable object** instead of throwing an error (e.g. `throw new Error('...')`). Uncaught errors in Server Actions are considered to be an internal server error and the error message [will not be sent to the client](/docs/app/building-your-application/routing/error-handling#securing-sensitive-error-information) in production to avoid leaking sensitive information. See more information on [error handling](#error-handling) in Server Actions.
```